packages/apps/tools/TESTPLAN.md

```diff
@@ -0,0 +1,83 @@
+# Test Scenarios
+
+## 1. Introduction
+
+### 1.1 Purpose
+The purpose of this document is to outline the test scenarios for the Tools Website following the Scrum framework.
+
+### 1.2 Scope
+This document covers high-level test scenarios for Tools Website and is intended to guide the creation of detailed test cases.
+
+## 2. User Stories and Acceptance Criteria
+
+- Home page
+  - All elements are correctly displayed on the home page and users can interact with them
+- Faucet
+  - Users can create new accounts by adding one or multiple public keys and fund 100 coins on all chains on any chain on Testnet.
+  - Users can fund existing accounts with 100 coins on any chain on Testnet.
+
+## 3. Test Scenarios
+
+### 3.1 [Home Page/All elements are visible]
+
+#### Scenario 1: On first page load the user can see all elements: Header placed at the top of the page, Side-menu that is initially opened with all available menu options, Right Drawer that is initially closed and opens on clicking the icon, and the main info content is displayed on the page
```

Comment:
I'd test this manually, it is far quicker to see if the criteria for this tests are met by looking at the UI ourselves.

Reply:
If this is going to be tested manually, should I remove this from here?